shopfloor/actions/message.py

```diff
@@ -818,8 +818,8 @@ def unable_to_pick_qty(self):
     def lot_replaced_by_lot(self, old_lot, new_lot):
         return {
             "message_type": "success",
-            "body": _("Lot %(old_lot_name)s replaced by lot %(new_lot_name)s.").format(
-                old_lot.name, new_lot.name
+            "body": _("Lot {old_lot_name} replaced by lot {new_lot_name}.").format(
+                old_lot_name=old_lot.name, new_lot_name=new_lot.name
```

Comment:
```suggestion
            "body": _(
                "Lot %(old_lot_name)s replaced by lot %(new_lot_name)s.",
                old_lot_name=old_lot.name,
                new_lot_name=new_lot.name,
```
Please use the builtin args/kwargs.

Reply:
I would also use `self.env._` instead of `_`, but that's a tiny detail, not blocking.

Reply:
> I would also use `self.env._` instead of `_`, but that's a tiny detail, not blocking.

This is a component, there's no `self.env` here.

Reply:
Right, sorry, I missed that.

Reply:
We do have an env on components. The main reason to keep them this way is to not diverge too much from 16.0.
That's why I've not suggested to do so :wink: